Vladyslav_Gurevych/HW8/MINorMAX.py

```diff
@@ -0,0 +1,44 @@
+def own_max(data: list, amount_of_result=1):
```

Comment:
annotations for amount_of_result.
amount_of_result: int = 1

and for return statment

Reply:
+